Add config docs about environment variables (#39)
OpenSpace/OpenSpace-Docs: using-openspace/configuration/index.md

# Configuration

OpenSpace uses a configuration file which is called `openspace.cfg`. This file is located in the OpenSpace folder. This file determines:

> - The SGCT configuration (the type of projection and number of windows)

Reviewer: How about removing the empty line after each point? makes no difference in the output 

Also, Mr @alexanderbock usually wants the list indented one step, so I guess that should be done for this file as well (?)

- The profile set on startup

- The paths where OpenSpace can find files

> - The visibility level for properties

Reviewer 2: ```suggestion
- The default visibility level for properties
```

- Configurations for modules

- Fonts and font sizes

- Log and script log outputs

> ...and a number of other configurations. For complete documentation with regards to the config file, please refer to [this page](#core_configuration).

Reviewer: This link does not work.

Reviewer: Interesting. Did not work in my local build

Feel free to ignore then :)

> ## Environment variables

Reviewer: ```suggestion
## Environment Variables
```

> Some of the configurations in the config file can use environment variables. These can be very handy if you use OpenSpace a lot, as they will speed up and / or customize your setup automatically. The environment variables are set as Path variables in a Windows environment. The environment variables are:

Reviewer: ```suggestion
Some of the configurations in the config file can use environment variables. These can be very handy if you use OpenSpace a lot, as they will speed up and/or customize your setup automatically. The environment variables are set as Path variables in a Windows environment. The environment variables are:
```

> - `OPENSPACE_SYNC` The sync folder is where OpenSpace stores all downloaded data, such as models, maps, textures, etc. If this environment variable is set, the sync folder which normally is placed inside the OpenSpace folder, can be placed anywhere on your computer. This is very handy if you are using multiple versions of OpenSpace, for the following reasons:

Reviewer 2: ```suggestion
- `OPENSPACE_SYNC`: The `sync` folder is where OpenSpace stores all downloaded data, such as models, maps, textures, etc. If this environment variable is set, the `sync` folder which normally is placed inside the OpenSpace folder, can be placed anywhere on your computer. This is very handy if you are using multiple versions of OpenSpace, for the following reasons:
```

Reviewer: Extending the suggestion above
```suggestion
- `OPENSPACE_SYNC`: The `sync` folder is where OpenSpace stores all downloaded data, such as models, maps, textures, etc. If this environment variable is set, the `sync` folder (which normally is placed inside the OpenSpace folder) can be placed anywhere on your computer. This is very handy if you are using multiple versions of OpenSpace, for the following reasons:
```

>   - If you get a fresh build of OpenSpace you don’t need to download the data again, if you already have the data on disk. This will save you time on startup.

Reviewer 2: ```suggestion
  - If you get a fresh build of OpenSpace you don't need to download the data again, if you already have the data on disk. This will save you time on startup.
```

>   - If you have multiple instances of OpenSpace, the required storage space on your disk will lessen as you only need to store the data once.

Reviewer 2: ```suggestion
  - If you have multiple instances of OpenSpace, the required storage space on your disk will lessen as you only need to store the data once as the `sync` folder can be reused across multiple OpenSpace versions.
```

> - `OPENSPACE_USER` The user folder is meant to be used for all user created files. These include webpanels, bookmarks, recordings, screenshots, data, and configs. If this environment variable is set, the user folder which normally is placed inside the OpenSpace folder, can be placed anywhere on your computer. This is very handy if you are using multiple versions of OpenSpace, for the following reasons:

Reviewer 2: ```suggestion
- `OPENSPACE_USER`: The `user` folder is meant to be used for all user created files. These include webpanels, bookmarks, recordings, screenshots, data, and configs. If this environment variable is set, the `user` folder which normally is placed inside the OpenSpace folder, can be placed anywhere on your computer. This is very handy if you are using multiple versions of OpenSpace, for the following reasons:
```

> - `OPENSPACE_USER` The user folder is meant to be used for all user created files. These include webpanels, bookmarks, recordings, screenshots, data, and configs. If this environment variable is set, the user folder which normally is placed inside the OpenSpace folder, can be placed anywhere on your computer. This is very handy if you are using multiple versions of OpenSpace, for the following reasons:

Reviewer: Just a note: we should make a separate page the user folder and how to use it. Would have bene nice to link to it here :)

>   - You can have one “global” folder for all your OpenSpace data, and all your OpenSpace instances will find it.

Reviewer 2: ```suggestion
  - You can have one "global" folder for all your OpenSpace data, and all your OpenSpace instances will find it.
```

>   - It might save you space as you don’t have to store data in multiple places.

Reviewer 2: ```suggestion
  - It might save you space as you don't have to store data in multiple places.
```

> - `OPENSPACE_GLOBEBROWSING` If this environment variable is set, the OpenSpaceData folder which normally is placed one level above the OpenSpace folder, can be placed anywhere on your computer. This folder stored all the map data from planets. This is very handy if you are using multiple versions of OpenSpace, for the following reasons:

Reviewer 2: ```suggestion
- `OPENSPACE_GLOBEBROWSING`: If this environment variable is set, the `OpenSpaceData` folder which normally is placed one level above the OpenSpace folder, can be placed anywhere on your computer. This folder stored all the map data from planets. This is very handy if you are using multiple versions of OpenSpace, for the following reasons:
```

Reviewer: ```suggestion
- `OPENSPACE_GLOBEBROWSING`: If this environment variable is set, the `OpenSpaceData` folder (which normally is placed one level above the OpenSpace folder) can be placed anywhere on your computer. This folder stores all the map data from planets. This is very handy if you are using multiple versions of OpenSpace, for the following reasons:
```

A bit unsure about the formulation "This folder stores all the map data form planets".  Not obvious to me what that means. And it's not all the data, right?

Author: Any suggestions how to make this clearer?

  - Faster loading of map tiles that you have already visited.

  - Less storage space as all instances of OpenSpace will use the same data.

> - `OPENSPACE_LEVEL` If this environment variable is set, the Property Visibility level will be determined on startup. The property visibility level is a setting that determines how complex the user interface should be. If you select `User`, for example, it means you won't see the more advanced properties that will be visible if you select `AdvancedUser`. The levels are:

Reviewer 2: ```suggestion
- `OPENSPACE_LEVEL`: If this environment variable is set, the Property Visibility level will be determined on startup. The property visibility level is a setting that determines how complex the user interface should be. If you select `User`, for example, it means you won't see the more advanced properties that will be visible if you select `AdvancedUser`. The levels are:
```

Reviewer: ```suggestion
- `OPENSPACE_LEVEL`: If this environment variable is set, the Property Visibility level will be determined on startup. The property visibility level is a setting that determines how complex the user interface should be. If you select `User`, for example, it means you won't see the more advanced properties that will be visible if you select `AdvancedUser`. The available levels in order of increasing complexity are:
```

>   - `NoviceUser`
> 
>   - `User`
> 
>   - `AdvancedUser`
> 
>   - `Developer`

Reviewer: ```suggestion
  - `NoviceUser`
  - `User`
  - `AdvancedUser`
  - `Developer`
```
